pola/management/commands/import_brands.py

```diff
@@ -0,0 +1,86 @@
+import csv
+import re
+
+from django.core.management import BaseCommand
+
+from company.models import Brand, Company
+from product.models import Product
+
+
+def update_product(self, brand, ean_code, company, product_name):
+    if Product.objects.filter(code=ean_code).count():
+        product = Product.objects.get(code=ean_code)
+        product.brand = brand
+        product.name = product_name
+        product.save()
+        self.stdout.write(self.style.SUCCESS(f"Successfully updated product {product_name}"))
+    else:
+        Product(company=company, brand=brand, code=ean_code, name=product_name).save()
+        self.stdout.write(self.style.SUCCESS(f"Successfully created product {product_name}"))
+
+
+class Command(BaseCommand):
+    help = 'Import lidl companies from .tsv file'
+
+    def add_arguments(self, parser):
+        parser.add_argument('tsv_filepath')
+
+    def handle(self, *args, **options):
+        if options['tsv_filepath']:
```

Comment:
```suggestion
        if not options['tsv_filepath'] or not os.path.exists(options['tsv_filepath']):
            raise SystemExit("Invalid path")
```
To będzie czytelniejsze, jeśli odwrócimy warunek